content-review/api/src/main/resources/ContentReviewItem.hbm.xml

```diff
@@ -26,5 +26,11 @@
         <property     name="retryCount"         type="long"                 not-null="false" />
         <property     name="nextRetryTime"      type="java.util.Date"       not-null="true" />
         <property     name="errorCode"          type="integer"              not-null="false" />
+
+        <joined-subclass name="org.sakaiproject.contentreview.dao.ContentReviewItemUrkund" table="CONTENTREVIEW_ITEM_URKUND">
```

Comment:
Instead of creating a new table with some specific items for Urkund's review maybe these should just be added as a properties to the ContentReviewItem? This way we don't get into the business of creating tables specific to any Content Review Impl, and if another impl has its own set of data it would like to persist this will work for those as well. Thoughts?